lib/Agrammon/DB/Dataset.pm6

```diff
@@ -687,4 +716,8 @@ class Agrammon::DB::Dataset does Agrammon::DB {
         return $data;
     }
 
+    method !variant {
+        return %!agrammon-variant<version>, %!agrammon-variant<gui>, %!agrammon-variant<model>;
```

Comment:
Can be `%!agrammon-variant<version gui model>` instead

Reply:
Done.